app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,48 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.all
+  end
+
+  def show
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def new
+    # @merchant = Merchant.new
+  end
+
+
+  def destroy
+    @merchant = Merchant.find(params[:id])
+
+    @merchant.destroy
+    redirect_to "/merchants"
+  end
+
+  def edit
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update
+    @merchant = Merchant.find(params[:id])
+
+    @merchant.update(merchant_params)
+    redirect_to "/merchants/#{@merchant.id}"
+  end
+
+  def create
+    merchant = Merchant.create!(merchant_params)
+    redirect_to "/merchants"
+  end
+
+  # def item_index
+  #   @merchant = Merchant.find(params[:id])
+  #   @merchant.items
+  # end
```

Comment:
Since you took this into account with the items controller index method, take out this commented code since you don't need it anymore.